.version

```diff
@@ -0,0 +1 @@
+3.1.3
```

Comment:
Can we support dev and stable versions with bumpversion?

Reply:
So I guess the question is what is the use-case for the "dev" version? If not in the phar context add  -dev to the version?

Reply:
If someone hand-builds a phar, though, then the version should still be reported as "dev". The use case is to know without question that, if a terminus build reports that it is stable, that it actually is the released version.

Reply:
The `self-update` library, which Terminus is already uses, has a [release](https://github.com/consolidation/self-update/blob/main/scripts/release) script that handles version-bumping. I'm not sure that I am still fond of version bump and release being an atomic operation, but if bumpversion doesn't do dev releases, maybe we can make our own.

Reply:
I used to use `release` to release my oss projects, but I am not using it any more, I just release manually.

Reply:
My point is that we just don't circulate development releases to customers in the same way we used to. At least we haven't in the last year or so since we've hardened and automated the release process.

Reply:
If someone installs Terminus via `git clone` and Support asks them to report their Terminus version, then `terminus --version` must return `-dev`.

Reply:
So what if the only way it gets a non-dev nomenclature is if it's built by GitHub actions?

Reply:
That would be fine, but I think that the GitHub action should be triggered on someone pushing a tag. We don't want GitHub actions to unintentionally make releases.

Reply:
They have to create a release from a pushed tag. Pushed tags happen when there's a successful merge of a pull request.